File: src/removeduplicatesfromsortedarray/Driver.java
package removeduplicatesfromsortedarray;

public class Driver {

    public static void main(String[] args) {
// Input: nums = []
// Output: 5, nums = [0,1,2,3,4,_,_,_,_,_]
        int[] nums = {0,0,1,1,1,2,2,3,3,4};
        Solution sol = new Solution();

        int k = sol.removeDuplicates(nums);
        System.out.println(java.util.Arrays.toString(nums));
        System.out.println(k);

    }

}

File: remove-duplicates-from-sorted-array-2/cpp/Driver.cpp
#include "Solution.h"

#include <vector>
#include <iostream>

int main() {
//   Input: nums = [1,1,1,2,2,3]
// Output: 5, nums = [1,1,2,2,3,_]
  std::vector<int> nums = {1,1,1,2,2,3};

  Solution sol;
  int k = sol.removeDuplicates(nums);
  std::cout << "The elements that are unique are: " << k << "\n";
  for (int el: nums) {
    std::cout << el << " ";
  }

  return 0;
}